src/test/javascript/spec/component/modeling-assessment-dashboard/modeling-assessment-dashboard.component.spec.ts

```diff
@@ -116,9 +116,9 @@ describe('ModelingAssessmentDashboardComponent', () => {
 
     it('should get Submissions', () => {
         // test getSubmissions
-        const modelingSubmissionServiceSpy = spyOn(modelingSubmissionService, 'getModelingSubmissionsForExerciseByCorrectionRound').and.returnValue(
-            of(new HttpResponse({ body: [modelingSubmission] })),
-        );
+        const modelingSubmissionServiceSpy = jest
+            .spyOn(modelingSubmissionService, 'getModelingSubmissionsForExerciseByCorrectionRound')
+            .mockReturnValue(of(new HttpResponse({ body: [modelingSubmission] })));
```

Comment:
In previous files, you specified the HttpResponse more. Can this be done here as well?

Probably `HttpResponse<Submission>(...)`?

Reply:
In fact, not. The modelingSubmission does not have a type. Refactoring these tests (and their equivalents for other exercise types) sounds good, but out of scope for this pr.